### PR TITLE
LCH-5024: Implement new interest list endpoint.

### PR DESCRIPTION
**Pull request by saxumVermes**

### CODE REVIEW

**Review by jose-oliveira — APPROVED** [review]

File: src/ContentHubClient.php [hunk 1]
```diff
@@ -725,6 +711,25 @@ public function getWebhookStatus() {
...
+   *   The UUID of the webhook.
+   * @param array $uuids
+   *   Entity UUIDs to add to Interest List.
+   *
+   * @return \Psr\Http\Message\ResponseInterface
+   *   The response.
+   *
+   * @throws \GuzzleHttp\Exception\RequestException
+   */
+  public function addEntitiesToInterestList($webhook_uuid, array $uuids) {
```
Comment (ankitsingh0188) [inline src/ContentHubClient.php]: argument type and return type missing.

File: src/ContentHubClient.php [hunk 2]
```diff
@@ -742,7 +747,98 @@ public function getInterestsByWebhook($webhook_uuid) {
...
+   *
+   * @param string $uuid
+   *   Interest UUID.
+   * @param string $webhook_uuid
+   *   Webhook UUID.
+   *
+   * @return \Psr\Http\Message\ResponseInterface
+   *   Response.
+   */
+  public function deleteInterest($uuid, $webhook_uuid) {
```
Comment (ankitsingh0188) [inline src/ContentHubClient.php]: argument type & return type missing.

File: src/ContentHubClient.php [hunk 3]
```diff
@@ -742,7 +747,98 @@ public function getInterestsByWebhook($webhook_uuid) {
...
+
+  /**
+   * Get the settings that were used to instantiate this client.
+   *
+   * @return \Acquia\ContentHubClient\Settings
+   *   Settings object.
+   *
+   * @codeCoverageIgnore
+   */
+  public function getSettings() {
```
Comment (ankitsingh0188) [inline src/ContentHubClient.php]: return type missing.

**Review by ankitsingh0188 — CHANGES_REQUESTED** [review]
Left some minor comments, looks good otherwise.

**Review by ankitsingh0188 — APPROVED** [review]
Looks good to me!!

**Review by amangrover90 — APPROVED** [review]
Looks good to me!!

**Pull request merged**